components/event-bus/Gopkg.toml

```diff
@@ -5,10 +5,28 @@ required = [
   "github.com/onsi/gomega", # for test matchers
```

Comment:
Everything above that line is a `constraint`, _not_ a `requirment`. Lines 1 to 5 should be deleted.